### ISSUE TITLE
  ACCESS_REFUSED - Login was refused using authentication mechanism AMQPLAIN. For details see the broker logfile.  

### ISSUE THREAD

**Issue by o2-web** [vyuldashev/laravel-queue-rabbitmq#376]
laravel v5.5 and vyuldashev/laravel-queue-rabbitmq 5.5

```
      ACCESS_REFUSED - Login was refused using authentication mechanism AMQPLAIN. For details see the broker logfile.  
```

my config:
```
    'rabbitmq' => [

            'driver' => 'rabbitmq',
            'queue' => env('RABBITMQ_QUEUE', 'default'),
            'connection' => PhpAmqpLib\Connection\AMQPLazyConnection::class,

            'hosts' => [
                [
                    'host' => env('RABBITMQ_HOST', '168.119.***.***'),
                    'port' => env('RABBITMQ_PORT', 5672),
                    'user' => env('RABBITMQ_USER', 'borisbot'),
                    'password' => env('RABBITMQ_PASSWORD', '*************'),
                    'vhost' => env('RABBITMQ_VHOST', '/'),
                ],
            ],

            'options' => [
                'ssl_options' => [
                    'cafile' => env('RABBITMQ_SSL_CAFILE', null),
                    'local_cert' => env('RABBITMQ_SSL_LOCALCERT', null),
                    'local_key' => env('RABBITMQ_SSL_LOCALKEY', null),
                    'verify_peer' => env('RABBITMQ_SSL_VERIFY_PEER', true),
                    'passphrase' => env('RABBITMQ_SSL_PASSPHRASE', null),
                ],
            ],

            /*
             * Set to "horizon" if you wish to use Laravel Horizon.
             */
            'worker' => env('RABBITMQ_WORKER', 'default'),

        ],

```

but I can successfully log in to the management panel with these credentials

im connecting from local machine to remote rabbitmq-server

please note that the user is not a guest.

is it mandatory to need an ssl certificate?

what did I miss?

**Comment by bloomlive** [comment]
Does the user have access to specified vhost? Even `/` needs to be added manually and freshly added users do not have permissions to access anything.

**Comment by o2-web** [comment]
solved with version 6.0 and considering documentation for 6.0 https://github.com/vyuldashev/laravel-queue-rabbitmq/blob/v6.0/README.md

**Comment by SergioMiller** [comment]
php artisan config:clear

**Comment by fendi-idn** [comment]
> solved with version 6.0 and considering documentation for 6.0 https://github.com/vyuldashev/laravel-queue-rabbitmq/blob/v6.0/README.md

Thanks... 
I'm using `v6.0.3` and miss-configured the env.
I was using `RABBITMQ_USERNAME` instead of `RABBITMQ_LOGIN`.
Thank you so much. You saved my day... 😄
